### PR TITLE
chore(curriculum): validate registration form project

### LINKED ISSUE #46540: Html validation errors and warnings for 9 RWD practice projects
Related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/46463#issuecomment-1155180861

The final code of the projects has been validated with three different html validators (https://www.freeformatter.com/html-validator.html, https://appdevtools.com/html-validator, https://validator.w3.org/), resutling in these errors and warnings:

All 9 projects have the error `Consider adding a “lang” attribute to the “html” start tag to declare the language of this document.`, this can be resolved adding `land="en"` to the html tag
 - [x] learn-responsive-web-design-by-building-a-piano/
 - [x] learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/
 - [x] learn-intermediate-css-by-building-a-picasso-painting/
 - [x] learn-css-colors-by-building-a-set-of-colored-markers/
 - [x] learn-css-animation-by-building-a-ferris-wheel/
 - [x] learn-css-transforms-by-building-a-penguin/
 
These 3 have both the error written above and `The character encoding was not declared.`, this can be solved adding the `meta` tag with the `charset` attribute.
 - [x] learn-html-forms-by-building-a-registration-form/
 - [x] learn-html-by-building-a-cat-photo-app/
 - [x] learn-css-variables-by-building-a-city-skyline/

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46540

I decided to break the step into two, because this is still early and I don't want to overwhelm the campers

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62bc29efa036af159e1a1392.md
Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62bc29efa036af159e1a1392.md:10]:
```suggestion
Nest a self-closing `link` element within the `head` element. Give it a `rel` attribute with value of `stylesheet` and an `href` attribute with a value of `styles.css`.
```

**Review by majestic-owl448 — DISMISSED** [review]
I approve, see if you think the comments could improve the text.

**Review by majestic-owl448 — DISMISSED** [review]

**Comment by Sboonny** [comment]
running

> npm run update-step-titles

gave me this error

image

**Comment by Sboonny** [comment]
there were two duplicated `step 42` and `step 43`, I had to delete them and fix the meta 

> npm run update-step-titles 

runs without issue, but it won't hurt to double-check if I did something wrong

**Comment by Sboonny** [comment]
@lasjorg I have cleared tab issues 👍

**Review by moT01 — CHANGES_REQUESTED** [review]
Something isn't quite right here. So you added a new step 5 and deleted the old step 42? I don't think step 42 needed to be deleted - it adds a good amount of styling to the form. Also, the code from the deleted step is still in the seed of steps after it. If you checkout this PR and go from step 43 to 44, the look of the form changes a lot cause the seed code from the delete step is still on 44 - but the user never adds that code as far as I can tell.

**Comment by Sboonny** [comment]
@moT01 nice catch, I deleted a step after the merge conflict. but I can't recreate it `npm run lint:challenges` run with error, I will open a third pr

**Pull request closed without merging**
